### ISSUE TITLE
Your 'Click Here To Get Started' is throwing a 404

### ISSUE THREAD

**Issue by cwbuege** [volatiletech/authboss#315]
Looking through Oauth2 solutions to find some open sources ones, came across yours and tried your 'Click Here To Get Started' link.  Its throwing a 404.  Thought you'd want to know.

**Comment by ibraheemdev** [comment]
Yes, the docs site was just implemented, and has not been deployed yet. That should be fixed soon.

**Comment by aarondl** [comment]
@ibraheemdev is there some specific steps we need to be taking? I was under the assemption the merge was enough :p

**Comment by ibraheemdev** [comment]
It has to be deployed. [Github pages](https://docsify.js.org/#/deploy) is probably the easiest option.

**Comment by aarondl** [comment]
Exists now
https://volatiletech.github.io/authboss/#/

**Comment by ibraheemdev** [comment]
#316
